### ISSUE TITLE
wav2lip_train loss with syncnet_wt? 

### ISSUE THREAD

**Issue by rizwanishaq** [Rudrabha/Wav2Lip#205]
In wav2lip_train the loss is 
loss = hparams.syncnet_wt * sync_loss + (1 - hparams.syncnet_wt) * l1loss
and in hparams syncnet_wt initially set to 0.0, and then it is set to 0.03 after how many epochs???
"# is initially zero, will be set automatically to 0.03 later. Leads to faster convergence."

It is seems it is set to 0.01 when it is average_sync_loss is < 0.75?

**Comment by prajwalkr** [comment]
> It is seems it is set to 0.01 when it is average_sync_loss is < 0.75?

yes
